tensorboard/webapp/widgets/histogram/histogram_test.ts

```diff
@@ -906,6 +906,35 @@ describe('histogram test', () => {
         fixture.detectChanges();
         expect(doHistogramsHaveColor(fixture)).toEqual([false, false, false]);
       });
+
+      it('puts color on histogram that mouse hovers over', () => {
```

Comment:
Do you need a test to ensure that mouseleave does not disable a histograms color when linked time is disabled?